### ISSUE TITLE
Pod in perpetual "Terminating" status on version 1.9.4-gke.1

### ISSUE THREAD

**Issue by enkodr** [kubernetes/kubernetes#61413]
/kind bug

**What happened**:
After upgrade to version 1.9.4-gke.1 on Google Cloud Platform, all of our pods that are supposed to be terminated keeps in a perpetual "Terminating" status:
```Killing container with id docker://app:Need to kill Pod```

**What you expected to happen**:
The pod should be terminated

**How to reproduce it (as minimally and precisely as possible)**:
Upgrade to version 1.9.4 and update an image or delete a pod associated with a deployment.

**Environment**:
- Kubernetes version: 
```Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.4", GitCommit:"bee2d1505c4fe820744d26d41ecd3fdd4a3d6546", GitTreeState:"clean", BuildDate:"2018-03-12T16:29:47Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9+", GitVersion:"v1.9.4-gke.1", GitCommit:"10e47a740d0036a4964280bd663c8500da58e3aa", GitTreeState:"clean", BuildDate:"2018-03-13T18:00:36Z", GoVersion:"go1.9.3b4", Compiler:"gc", Platform:"linux/amd64"}```
- Cloud provider: Google Cloud Platform

**Comment by liggitt** [comment]
can you provide the pod spec associated with the pod?

**Comment by enkodr** [comment]
Here it is, but this is happening with all of my pods.
Here is the deployment yaml used:
`apiVersion: apps/v1beta1
kind: Deployment
metadata:
  name: app
  labels:
    chart: "app"
spec:
  replicas: 1
  revisionHistoryLimit: 10
  minReadySeconds: 5
  strategy:
    type: RollingUpdate
    rollingUpdate:
      maxUnavailable: 0
  template:
    metadata:
      labels:
        role: app
        environment: test
    spec:
      terminationGracePeriodSeconds: 10
      containers:
        - image: eu.gcr.io/<id>/app:1.2.1
          name: app
          ports:
            - containerPort: 80
              name: app-http
            - containerPort: 443
              name: app-https
          volumeMounts:
            - name: nginx-config
              mountPath: /etc/nginx/conf.d/default.conf
              subPath: nginx.app.conf
            - name: tls-certificates
              mountPath: /etc/nginx/ssl
            - name: nginx-config
              mountPath: /etc/nginx/.htpasswd
              subPath: htaccess
      volumes:
        - name: nginx-config
          configMap:
            name: app-config
        - name: tls-certificates
          secret:
            secretName: tls-certificates`

**Comment by liggitt** [comment]
that looks like https://github.com/kubernetes/kubernetes/issues/61178, triggered by use of a subPath mount, fixed in 1.9.5, released yesterday

**Comment by enkodr** [comment]
Thanks. I'll keep an eye on it.
